corehq/apps/users/tests/test_signals.py

```diff
@@ -63,9 +65,18 @@ def test_webuser_save(self, _, send_to_es, invalidate,
 @patch('corehq.apps.callcenter.tasks.sync_usercases')
 @patch('corehq.apps.cachehq.signals.invalidate_document')
 @patch('corehq.apps.users.signals._should_sync_to_es', return_value=True)
-@es_test(requires=[user_adapter], setup_class=True)
+@es_test
```

Comment:
is this intentionally left un-migrated?

Reply:
Oops! Seems like a bad commit. Looks like I accidentally uncommitted what has already been done.
Thanks for pointing it. Good catch 💯

Reply:
https://github.com/dimagi/commcare-hq/pull/32522/commits/48ae8b9218d17f414c4a807545fb5baabb518887